Svc/StaticMemory/StaticMemoryComponentImpl.cpp

```diff
@@ -42,7 +42,10 @@ void StaticMemoryComponentImpl ::bufferDeallocate_handler(const NATIVE_INT_TYPE
     FW_ASSERT(m_allocated[portNum], portNum); // It is also an error to deallocate before returning
     // Check the memory returned is within the region
     FW_ASSERT(fwBuffer.getData() >= m_static_memory[portNum]);
-    FW_ASSERT((fwBuffer.getData() + fwBuffer.getSize()) <= (m_static_memory[portNum] + sizeof(m_static_memory[0])), fwBuffer.getSize(), sizeof(m_static_memory[0]));
+    FW_ASSERT(
+        (fwBuffer.getData() + fwBuffer.getSize()) <= (m_static_memory[portNum] + sizeof(m_static_memory[0])),
+        static_cast<FwAssertArgType>(fwBuffer.getSize()),
+        sizeof(m_static_memory[0]));
```

Comment:
I think this line needs a static cast to `FwAssertArgType`

Reply:
Fixed